### PR TITLE
Allow logging to TensorBoard (#6)

### LINKED ISSUE #6: Make the agent produce log files suitable for TensorBoard
* https://www.tensorflow.org/guide/summaries_and_tensorboard
* https://github.com/neka-nat/tensorboard-chainer
* https://github.com/crowdAI/marLo/wiki/Experiments-and-plotting-with-TensorBoard#tensorboard

### PR DESCRIPTION
**Pull request by bakoe**
Closes #6

### CODE REVIEW

File: train_agent.py [hunk 1]
```diff
@@ -63,16 +65,63 @@ def main(steps, gpu, imagefile, boxfile):
...
+        debug_image = Image.new(mode='RGB',
+                                size=(image.width, image.height + 16),
+                                color=0)
+        debug_image.paste(image)
+
+        draw = ImageDraw.Draw(debug_image)
+        draw.rectangle(bbox.tolist(), outline=(255, 255, 255))
+        draw.text(xy=(2, image.height + 2),
+                  text='Last action: %s' % last_action_name,
+                  fill='white')
```
Comment (bakoe) [inline train_agent.py]: @bakoe Add font and font size to avoid possible errors caused by different/changing defaults.

File: train_agent.py [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
+from tb_chainer import SummaryWriter
+import time
 
 @click.command()
 @click.option("--steps", "-s", default=2000, help="Amount of steps to train the agent.")
 @click.option("--gpu/--no-gpu", default=False)
 @click.option("--imagefile", "-i", default='image_locations.txt', help="Path to the file containing the image locations.", type=click.Path(exists=True))
 @click.option("--boxfile", "-b", default='bounding_boxes.npy', help="Path to the bounding boxes.", type=click.Path(exists=True))
-def main(steps, gpu, imagefile, boxfile):
+@click.option("--tensorboard/--no-tensorboard", default=True)
```
Comment (bakoe) [inline train_agent.py]: @bakoe Add instructions to README.md on how to display the results using TensorBoard

**Review by J-L-O — APPROVED** [review]
Cool stuff! Besides the things you already noted yourself, please also link the tb_chainer package in the readme. Also, I found that tensorflow doesn't work with Python 3.7 as of yet, so unless it worked for you please mention that as well to avoid confusion.

**Comment by bakoe** [comment]
@J-L-O Thanks for your review and for your comments! I'll definitely do that.

> Also, I found that tensorflow doesn't work with Python 3.7 as of yet, so unless it worked for you please mention that as well to avoid confusion.

Actually I got Tensorflow (+ TensorBoard) to work on macOS with Python 3.7 by manually installing the appropriate pip package:

```bash
pip install --upgrade https://storage.googleapis.com/tensorflow/mac/cpu/tensorflow-1.12.0-py3-none-any.whl
```

This was taken from https://github.com/tensorflow/tensorflow/issues/20444#issuecomment-442767411. I'll include corresponding remarks and links to the README.md as well. Thanks for noticing!

**Pull request merged**